src/applications/vaos/unenrolled-vaccine/components/FormLayout.jsx

```diff
@@ -13,7 +13,7 @@ export default function FormLayout({ children }) {
   return (
     <div className="vads-l-grid-container vads-u-padding-x--2p5 large-screen:vads-u-padding-x--0 vads-u-padding-bottom--8">
       <Breadcrumbs>
-        <Link to={match.url}>COVID-19 vaccine</Link>
+        <va-link href={match.url} text="COVID-19 vaccine" />
```

Comment:
This link is going to a page that doesn't exist. Suggest reverting back the code and making the fix in the breadcrumbs PR.

Reply:
Reverted this